src/schema/schema.ts

```diff
@@ -196,8 +197,8 @@ export function createZimbraSchema(
 					),
 				createMountpoint: (_, variables) =>
 					client.createMountpoint(variables as CreateMountpointInput),
-				deleteAppointment: (_, { inviteId }, { zimbra }) =>
-					zimbra.appointments.delete({ inviteId }),
```

Comment:
please remove previous method declaration from zimbra-client/index.js

Reply:
@silentsakky 
If my understanding is correct I need to remove the delete method from api.appointments, from zm-x-web\src\lib\zimbra-client\index.js.
Please confirm

Reply:
correct

Reply:
Fixed and pushed.